homeTask70/HomeTask70/ScreenShot.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using OpenQA.Selenium;
+using System.IO;
+
+
+namespace homeTask70
+{
+    public abstract class ScreenShot
+    {
+        public static void TakeScreenshot(IWebDriver Driver)
+        {
+            try
+            {
+                Screenshot ss = ((ITakesScreenshot)Driver).GetScreenshot();
+                string currentTime = DateTime.Now.ToString("dd_MM_yy_HH_mm_ss");
+                string path = Directory.GetCurrentDirectory().Replace(Directory.GetCurrentDirectory(), @"C:\Users\max\Git\newGitFolder\homeTask70\images\");
```

Comment:
Please, use here this way to set a path of screenshots saving - https://stackoverflow.com/questions/816566/how-do-you-get-the-current-project-directory-from-c-sharp-code-when-creating-a-c. We shouldn't leave local paths within our projects at all. In case if I will download your repository and try to run tests this method will fail.

Reply:
Done